clients/metadata/metadata_cassandra.go

```diff
@@ -1840,6 +1843,68 @@ func (s *CassandraMetadataService) DeleteConsumerGroup(ctx thrift.Context, reque
 	return nil
 }
 
+// DeleteConsumerGroupUUID deletes the consumer-group corresponding to the given UUID
+// from the consumer_groups table
+func (s *CassandraMetadataService) DeleteConsumerGroupUUID(ctx thrift.Context, request *m.DeleteConsumerGroupUUIDRequest) error {
```

Comment:
so dumb question, why this new API is needed?

Reply:
Since the "DeleteConsumerGroup" API now only moves the CG to deleting state, this new API is needed to move the CG to _deleted_ state (and apply TTL to the row in the consumer_groups table, etc). Basically this would do the actual cleanup of the row .. and would be called by retention manager when the CG is ready to be actually purged.